src/identities.js

```diff
@@ -64,15 +70,47 @@ class Identities {
   }
 
   async verifyIdentity (identity) {
-    const verified = await this._keystore.verify(
+    if (!Identity.isIdentity(identity)) {
+      return false
+    }
+
+    const knownID = this._knownIdentities.get(identity.signatures.id)
+    if (knownID) {
+      return identity.id === knownID.id &&
+             identity.publicKey === knownID.publicKey &&
+             identity.signatures.id === knownID.signatures.id &&
+             identity.signatures.publicKey === knownID.signatures.publicKey
+    }
+
+    const verifyIdSig = await Keystore.verify(
```

Comment:
Which `Keystore` instance does this use? Shouldn't this be `this._keystore.verify()`?